tests/application_tests/src/test_report.py

```diff
@@ -88,3 +90,28 @@ def test_add_report(self):
         nr_reports = len(self.driver.find_elements(By.CLASS_NAME, "card"))
         self.driver.find_element(By.CLASS_NAME, "button.primary").click()
         self.wait.until(nr_elements((By.CLASS_NAME, "card"), nr_reports + 1))
+
+    def test_report_axe_accessibility(self):
+        """Run aXe accessibility check on a report."""
+        axe = Axe(self.driver)
+        axe.inject()
+
+        # Analyze report page
+        report = self.driver.find_elements(By.CLASS_NAME, "card")[1]
+        report.click()
+        results1 = axe.run()
+
+        # Process aXe results
+        violation_results = results1["violations"]
+        axe.write_results(results1, '../../build/a11y.json')
+        readable_report = axe.report(violation_results)
+        filename = pathlib.Path('../../build/a11y_violations.txt')
+        try:
+            with open(filename, "w", encoding="utf8") as report_file:
+                report_file.write(readable_report)
+        except OSError:
+            print("Could not write axe violations txt")
+
+        # If there are violations, output the readable report data
+        # TODO - enable in https://github.com/ICTU/quality-time/issues/6354
+        # self.assertEqual(0, len(violation_results), readable_report)
```

Comment:
Can we maybe add an assert that the number of violations is <= the current number of violations?